app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1>Task List</h1>
+<% if @tasks.count > 0 %>
+    <ul>
+      <% @tasks.each do |task| %>
+
+        <% if !(task.completed_at.nil?) %>
+          <h5><%= link_to task.name, task_path(task) %></h5>
+          ✓ Completed on: <%= task.completed_at %>
+          <%= link_to "Undo", complete_task_path(task), method: :patch %>
+        <% elsif task.completed_at.nil? %>
+          <h5><%= link_to task.name, task_path(task) %></h5>
+          <%= link_to "☐ Mark done!", complete_task_path(task), method: :patch %>
+        <% end %>
+
+        <li>
+          Description: <%= task.description %>
+          <%= link_to "Edit", edit_task_path(task) %>
+          <%= link_to "Delete", task_path(task), method: :delete, data: {confirm: "Are you sure you want to delete the \"#{task.name}\" task?"} %>
+          <br>
+          <br>
+        </li>
+      <% end %>
+      <%= link_to "Add new task", new_task_path %>
+    </ul>
+  <% else %>
+    <h3>No tasks in the system<h3>
+  <% end %>
```

Comment:
Okay, this is a small thing, but it has big implications!

You left this `link_to` inside the positive case of the `if` block!~ Which means, if there are no tasks already in the server's DB, then there is no way to add them! 😱 This is a good reminder that Tests Don't Catch Everything, and that clicking thru a website and trying ALLL the features is important!
```suggestion
      <% end %>
    </ul>
  <% else %>
    <h3>No tasks in the system<h3>
  <% end %>
  <%= link_to "Add new task", new_task_path %>
```